src/main/java/uk/gov/pay/adminusers/queue/event/EventMessageHandler.java

```diff
@@ -44,7 +44,7 @@
     private final ServiceFinder serviceFinder;
     private final UserServices userServices;
     private final ObjectMapper objectMapper;
-    private final DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ofPattern("dd, LLLL yyyy"); // 09, March 2022
+    private final DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ofPattern("dd LLLL yyyy"); // 09 March 2022
```

Comment:
I think the more familiar `MMMM` would give the same output as `LLLL`. I think we also want `d` rather than `dd` because we do not want a leading zero.

Reply:
thanks, will update

Reply:
Thanks, I've made both those changes